src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
 
 If you can’t find answers to your questions in the docs, search for your answer or ask questions in our [Stellar Developer Discord](https://discord.gg/stellardev) or [Stack Exchange](https://stellar.stackexchange.com/).
 
-<WayfindingBoxes />
+## Stellar wayfinding based on audience type
+
+:::note
+
+There's much more you can do with the Stellar network than what's listed below! These pathfinding guides are meant to give you an idea of how to get started navigating the docs to find necessary conceptual information, tutorials, data availability recommendations, and more based on what you're trying to accomplish.
+
+:::
+
+### Stellar for asset issuers
+
+| **Stellar Assets** |  |  |
+| :-: | :-- | :-- |
+| **Learn** | [Asset and token introduction](https://developers.stellar.org/docs/tokens) | Learn about the difference between assets issued on Stellar and custom smart contract tokens |
+| **Learn** | [Data availability overview](https://developers.stellar.org/docs/data) | Learn about the different mechanisms for data availability on Stellar |
+| **Guide** | [Quickstart script](https://developers.stellar.org/docs/tokens/quickstart) | Issue an asset on the Stellar network in a single transaction |
+| **Learn** | [Asset design considerations](https://developers.stellar.org/docs/tokens/control-asset-access) | Learn about certain considerations (e.g., flags, asset naming, etc.) that are important when issuing an asset |
+| **Learn** | [Stellar Asset Contract (SAC)](https://developers.stellar.org/docs/tokens/stellar-asset-contract) | Learn about interacting with assets issued on Stellar in smart contracts |
+| **Guide** | [Deploy an SAC](https://developers.stellar.org/docs/tokens/stellar-asset-contract) | Deploy a Stellar Asset Contract with the Stellar CLI |
```

Comment:
It might be my ocd but is it possible to put the `Guides` together instead of split between `Learns`?

Reply:
I'm okay with your suggested formatting. I was thinking it worked sequentially for a user to follow the current path but I also think it could be confusing to have types of content separate from each other. I'll do as you suggest.

Reply:
Ah that makes sense. I'm happy with either format

Reply:
Reordered based on feedback!